src/circular-progress.jsx

```diff
@@ -144,13 +144,13 @@ const CircularProgress = React.createClass({
   _rotateWrapper(wrapper) {
     if (this.props.mode !== 'indeterminate') return;
 
-    AutoPrefix.set(wrapper.style, 'transform', 'rotate(0deg)');
```

Comment:
I haven't looked at how to remove `add` yet. I had some issue when trying to do so.
That's definitly something we should adresse.